data-plane-framework/draft-ietf-detnet-data-plane-framework.xml

```diff
@@ -635,6 +657,22 @@ CE1----EN1--------R1-------R2-------R3--------EN2-----CE2
      prevent misordering of packets. In both cases the protection paths are
      established and maintained for the duration of the DetNet service.
      </t>
+   </section>
+     <section title="Path Differential Delay">
+             <t>
+     In the preceding example, reordering of packets is dependent on the number
+     of out-of-order packets that can be buffered and the delay difference of
+     arriving packets.  DetNet uses configuration of the number of maximum
+     number out-of-order packets and the maximum delay that out-of-order
+     packets can be held before being delivered. If the differential delay
+     between paths is large enough or the arrival rate of packets large enough
+     packets may be dropped instead of being reordered.  Likewise elimination

```

Comment:
I think this is not precise. What this text says is that if POF buffer is too small (bad design!) then packet drops occur, what is trivial.
The prerequisite for POF is that the "differential delay" must be smaller than the "Maximum delay" minus the "fastest path delay of the packet". If this is fulfilled than POF buffer can be calculated. Arrival rate does matter from re-ordering perspective only for calculating how big buffers are needed.

Reply:
[Don] Again this is TSN derived text. ...

[Balazs] In that part of CB text is broken. :--((( CB does not really define details of a POF.